### DISCUSSION TITLE
2022-05-26 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#190] [Meetings]
#### Current list of units discussed and status of each, updated from 2022-04-28 meeting: [2022 UCUM Units proposed_2022-05-26 Update.xlsx](https://github.com/ucum-org/ucum/files/8758244/2022.UCUM.Units.proposed_2022-05-26.Update.xlsx)

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
2022-05-23, 1922HRS, ETA: AGENDA PENDING UPDATE. 

@ucum-org/advisors @ucum-org/regenstrief

**Comment by djvreeman** [comment]
Last time Regenstrief committed to proposing the process for adding members to the Board of Advisors at our next meeting. Will that be proposed on 5/26? We were ready to consider adding another well-qualified member and would greatly benefit from additional input/leadership, but were halted from that discussion.

**Comment by ehafeza** [comment]
Hi Dan,
Thank you for the reminder!
The governance discussion should be separate from  the content discussion which is why we tabled the topic for a future date.   We are planning a larger governance discussion for UCUM just like the one we are currently having regarding LOINC. Please know that we, including Marjorie our Executive Director are acutely aware of the need for additional and qualified members and are committed to ensuring we are successful in pursuing that goal.

Please feel free to reach out to me or Marjorie if you have additional questions or concerns.


Thank you much
Eza

Eza Hafeza
Director of Terminology Services and Operations
LOINC<https://loinc.org/>® and Health Data Standards

***@***.******@***.***>
Cell: (310) 961-7613


From: Daniel Vreeman ***@***.***>
Sent: Monday, May 23, 2022 5:58 PM
To: ucum-org/ucum ***@***.***>
Cc: Hafeza, Eza ***@***.***>; Team mention ***@***.***>
Subject: [External] Re: [ucum-org/ucum] 2022-05-26 Meeting Agenda (Discussion #190)

This message was sent from a non-IU address. Please exercise caution when clicking links or opening attachments from external sources.


Last time Regenstrief committed to proposing the process for adding members to the Board of Advisors at our next meeting. Will that be proposed on 5/26? We were ready to consider adding another well-qualified member and would greatly benefit from additional input/leadership, but were halted from that discussion.

**Comment by aflackey** [comment]
@ucum-org/advisors & @ucum-org/regenstrief 
The May meeting has been canceled. We will resume discussions in our next scheduled meeting, 16 June, 4:00 pm EDT.
